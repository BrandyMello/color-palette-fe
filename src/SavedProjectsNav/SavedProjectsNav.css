h2 {
  font-size: 34px;
}

.project_buttons {
  background-color:	#808080;
  border: 1px solid black;
  border-radius: 3px;
  color: white;
  font-size: 18px;
  font-weight: bold;
  margin: 5px 5px 20px;
  height: 46px;
  width: 170px;
}

.project_buttons:hover {
  background-color: #46a3e4;
}

.saved-section {
  border: 2px solid #808080;
  border-radius: 8px;
}