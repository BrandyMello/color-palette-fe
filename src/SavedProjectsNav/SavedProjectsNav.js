import React from 'react';
// import { NavLink } from 'react-router-dom';

// const goToProject = () => {

// }

const SavedProjectsNav = ({ projects }) => {
let project = projects.map((project, index) => <button key={index}>{project.name}</button>);
console.log(project.id)
  return (
    <div>
      { project }
    </div>
  )
}

export default SavedProjectsNav