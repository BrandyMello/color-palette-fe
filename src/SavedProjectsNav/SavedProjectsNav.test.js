import React from 'react';
import { shallow } from 'enzyme';
import SavedProjectsNav from './SavedProjectsNav';

describe('SavedProjectsNav', () => {
  it('should match snapshot', () => {
    const mockProjects = [{
      id: 1,
      name: 'Warm Colors'
    }, {
      id: 2,
      name: 'Cool Colors'
    }];

    const wrapper = shallow(<SavedProjectsNav
      projects={mockProjects}
    />);

    expect(wrapper).toMatchSnapshot();
  });
})