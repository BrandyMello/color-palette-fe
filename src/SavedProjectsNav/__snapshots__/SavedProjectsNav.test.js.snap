// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SavedProjectsNav should match snapshot 1`] = `
<div
  className="saved-section"
>
  <h2>
    Your Saved Projects
  </h2>
  <NavLink
    key="0"
    to="/projects/1"
  >
    <button
      className="project_buttons"
    >
      Warm Colors
    </button>
  </NavLink>
  <NavLink
    key="1"
    to="/projects/2"
  >
    <button
      className="project_buttons"
    >
      Cool Colors
    </button>
  </NavLink>
</div>
`;
